Parser/pegen.c

```diff
@@ -184,7 +184,7 @@ _PyPegen_get_expr_name(expr_ty e)
         case BoolOp_kind:
         case BinOp_kind:
         case UnaryOp_kind:
-            return "operator";
+            return "expression";
```

Comment:
+1, using "expression" here is indeed clearer than "operator" (despite seeming incorrect to the literal-minded :-).